api/src/test/java/com/flipkart/flux/api/WorkflowStateSummaryTest.java

```diff
@@ -10,24 +10,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.flipkart.flux.commons.dto;
+package com.flipkart.flux.api;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.core.Is;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
```

Comment:
In my view optimizing imports improves the code readability. That's why I always prefer to do it (lot of people prefer the same).
In this case it won't apply, but cases like importing .* creates unnecessary linkages at compilation time (If I remember correctly), It doesn't impact runtime performance though.